docs/docs/scanner/misconfiguration/custom/index.md

```diff
@@ -120,21 +119,50 @@ Trivy supports extra fields in the `custom` section as described below.
-| Field name                 | Allowed values                                                    |        Default value         |     In table     |     In JSON      |
-|----------------------------|-------------------------------------------------------------------|:----------------------------:|:----------------:|:----------------:|
-| title                      | Any characters                                                    |             N/A              | :material-check: | :material-check: |
-| description                | Any characters                                                    |                              | :material-close: | :material-check: |
-| schemas.input              | `schema["kubernetes"]`, `schema["dockerfile"]`, `schema["cloud"]` | (applied to all input types) | :material-close: | :material-close: |
-| custom.id                  | Any characters                                                    |             N/A              | :material-check: | :material-check: |
-| custom.severity            | `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`                               |           UNKNOWN            | :material-check: | :material-check: |
-| custom.recommended_actions | Any characters                                                    |                              | :material-close: | :material-check: |
-| custom.input.selector.type | Any item(s) in [this list][source-types]                          |                              | :material-close: | :material-check: |
-| url                        | Any characters                                                    |                              | :material-close: | :material-check: |
+| Field name                 | Allowed values                                                    | Description                                             |         Default value         |     In table     |     In JSON      |     Required in Trivy Check     |
+|----------------------------|-------------------------------------------------------------------|:-------------------------------------------------------:|:-----------------------------:|:----------------:|:----------------:| :-----------------------------: |
+| title                      | Any characters                                                    | Name of the policy                                      |              N/A              | :material-check: | :material-check: |     :material-check:            |
+| description                | Any characters                                                    | Description of the problem                              |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| schemas.input              | `schema["kubernetes"]`, `schema["dockerfile"]`, `schema["cloud"]` | Used to validate the policy for syntax errors           |  (applied to all input types) | :material-close: | :material-close: |     :material-check:            |
```

Comment:
Schema isn't required per se as it only is needed for validating the checks for functional correctness (syntax).

Furthermore, if a file has a file extension we expect scanning will still work just fine. For instance, if you try to scan argocd configs without having an argocd schema is still fine as they're treated as regular YAML files.

Reply:
Can we format this a little better, this is how it renders for me in the browser.
<img width="398" alt="image" src="https://github.com/aquasecurity/trivy/assets/1254783/3b5f7bee-7bd8-4331-aa2a-603279757717">